Trivago/Jade/Application/JsonApi/Request/RequestFactory.php

```diff
@@ -283,6 +298,21 @@ private function parseFilters(ResourceConfig $resourceConfig, CompositeFilter $c
         }
     }
 
+    /**
+     * Validating the "path" is not under "excluded_attributes" config section
+     * @param ResourceConfig $resourceConfig
+     * @param $key
+     * @param $path
+     */
+    private function validateExcludedPaths(ResourceConfig $resourceConfig, $key, $path)
+    {
+        if($this->strictFilteringAndSorting && $resourceConfig->isAttributeExcluded($resourceConfig->getRealPath($path))){
+            throw InvalidRequest::createWithMessage($key, 'invalid_path',
```

Comment:
Please put each argument in a new line and also use sprintf for the message to avoid opening and closing a string